docs/shared-content/octopus-recommendations/project-recommendations.include.md

```diff
@@ -0,0 +1,58 @@
+
+We built Octopus Deploy with the core concept of consistency across all environments.  The process used to deploy to your development environment is the same process used to deploy to your production environment. You can enable or disable specific steps, but it's the same process, and therefore the same parts, deployed to development or testing environments, that will make it to production.  The production deployment will be a non-event because you've tested the process many times, once for each environment in the lifecycle before production. 
+
+Knowing the underlying concept of Octopus Deploy is consistency; here are our project recommendations.
+
+## Keep projects simple
+
+One of our favorite programming maxims is the [single responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle). It says:
+
+> A class/function/method should do one thing, and it should do it well.  
+
+The same is true for projects.  Projects should be straightforward and should deploy a component of an application.  
```

Comment:
I wonder if this section should be qualified with some pros/cons of a single component project (e.g. microservices architectural style) vs a more combined project approach, even for octofx? 

Splitting it by component makes it easier to deploy the individual components, but makes it more annoying when deploying the whole thing - even with the deploy a release step.

It could even just a call-out to say:

> Whilst creating projects per component is a good practice, it's not the rule. Octopus provides flexibility to allow you to model your projects to meet your requirements.

Reply:
Good point, tweeked